docs/gateway.md

```diff
@@ -78,6 +78,71 @@ Options:
   -V, --version                    Print version information
 ```
 
+#### Gateway Peg-in Process
+This section outlines the steps to add a gateway to a federation and subsequently fund the gateway.
+
+##### Connecting the Gateway to a Federation
+Start by connecting your gateway to the desired federation using the following command:
+```bash
+$ gateway-cli connect-fed fed1xxxxxxx
+```
+##### Requesting a Deposit Address
+Once the gateway is successfully integrated into the federation, you can request a new address to deposit coins:
+```bash
+$ gateway-cli address --federation-id fed1xxxxxxx
```

Comment:
nit: The federation id is a hex string not the invite code beginnig with `fed1`. Technically `fed1` is also a hex string, but might be confusing.